docs/data/tree-view/rich-tree-view/customization/customization.md

```diff
@@ -0,0 +1,29 @@
+---
+productId: x-tree-view
+title: Rich Tree View - Customization
+components: RichTreeView, TreeItem
+packageName: '@mui/x-tree-view'
+githubLabel: 'component: tree view'
+waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/treeview/
+---
+
+# Rich Tree View - Customization
+
+<p class="description">Learn how to customize the rich version of the Tree View component.</p>
+
+## Basics
+
+### Custom label
+
+Use the `label` slot to customize the Tree Item label or to replace it with an entirely custom component.
+
+The `slotProps` prop allows you to pass props to the default label component:
+
+:::warning
+TODO
+:::
+
+The `slots` prop allows you to replace the default label with your own component.
```

Comment:
I added this demo here because it make no sense on the `SimpleTreeView`

I think it can be a good example for `slots.label` until we have a built-in edition